src/com/ms/silverking/cloud/dht/daemon/MessageModule.java

```diff
@@ -603,12 +604,12 @@ public PrimarySecondaryIPListPair getReplicaListPair(long context, DHTKey key, R
             List<IPAndPort>   replicaList;
             
             if (debug) {
-                System.out.println("\t\tgetReplicas "+ key +"\t"+ oqm);
+                Log.fineAsyncf("getReplicas %s \t %s",key , oqm);
             }
             // FUTURE - think about improvements
             replicaList = ringMaster.getReplicaList(key, oqm, ownerQueryOpType);
             if (debug) {
-                System.out.println("\t\t"+ key +"\t"+ CollectionUtil.toString(replicaList, ':'));
+                Log.fineAsyncf("%s \t %s",key , CollectionUtil.toString(replicaList, ':'));
```

Comment:
debug is off by default. Make Log.warningAsyncf so that the debug flag has an effect